activerecord/test/cases/schema_dumper_test.rb

```diff
@@ -193,6 +193,15 @@ def test_schema_dumps_partial_indices
     end
   end
 
+  def test_schema_dumps_index_included_columns
+    index_definition = dump_table_schema("companies").split(/\n/).grep(/t\.index.*company_include_index/).first.strip
+    if ActiveRecord::Base.connection.supports_index_include?
+      assert_equal 't.index ["firm_id", "type"], name: "company_include_index", include: [:name, :account_id]', index_definition
+    else
+      assert_equal 't.index ["firm_id", "type"], name: "company_include_index"', index_definition
+    end
+  end
```

Comment:
I think this test should be moved to activerecord/test//cases/adapters/postgresql/schema_test.rb after moving the schema change below. This feature is postgres specific so it should be contained to postgres files as much as possible.